src/vcgen.cpp

```diff
@@ -670,6 +670,34 @@ optional<string> encodeOp(State &st, mlir::ConstantOp op) {
 
     st.regs.add(op, Integer(i.getSExtValue(), bw));
     return {};
+  } else if (mlir::SparseElementsAttr sparseAttr = attr.dyn_cast<mlir::SparseElementsAttr>()) {
+    std::vector<z3::expr> sparse_values;
```

Comment:
I think it's good to follow the same c++ naming convention as MLIR code. (i.e use camelcase rather than snake case)

Reply:
Okay~

Reply:
Please check!
I resolved it.
Thanks!